privacy.adoc

```diff
@@ -363,13 +363,13 @@ public keys. A public key is a pseudonym for a user, and a user can
 have multiple pseudonyms. In an ideal world, your in-person identity
 is decoupled from your Bitcoin pseudonyms. Unfortunately, due to the
 privacy problems described in this chapter, this decoupling usually
-degrade over time.
+degrades over time.
 
 To mitigate the risks of having your personal data revealed is to not
-give it out in the first place or not give it to centralized services
-that <<kycdbs,build big databases>> that can leak. An article by
+give it out in the first place nor to give it to centralized services,
+which <<kycdbs,build big databases>> that can leak. An article by
```

Comment:
If we go for "technical" xrefs, we should rewrite this as 

`centralized services, which build big databases (see <<nonblockchainprivacy>>) that can ...`

or something similar. Note how we can't have a technical xref to kycdbs, which is just a reference to a paragraph inside "Non-blockchain privacy" section, and that paragraph has no number or name.

Reply:
I would write (see <<nonblockchainprivacy>>) as suggested by you. After all, it is a pretty small section and we don't risk the reader not finding the information she's lookinf for